qutebrowser/browser/webelem.py

```diff
@@ -173,6 +173,10 @@ def is_content_editable(self) -> bool:
         except KeyError:
             return False
 
+    def is_content_editable_prop(self) -> bool:
```

Comment:
There is a `is_content_editable()` method just above this, what's the difference?

Reply:
Maybe I should document this a better. 

(apparently) js has two ways of getting the 'contenteditable' status of an element. The first is getting the actual property (`elem.contenteditable`). This works, but it does not handle inheritance well (the property won't be on child elements) and contentedness may be set to many things.

The second way is `elem.isContentEditable` which does handle inheritance properly. So this property will return true for any child of a contenteditable element (which is what makes it work). Unfortunately, this won't work everywhere, as we need the actual contenteditable mode to work in strict mode (I think), so that's why I split it into two methods.